tests/test_api.py

```diff
@@ -32,66 +32,93 @@ def test_netcdf_extract_bbox():
 def test_kml_extract_bbox():
     result = geoextent.fromFile("tests/testdata/kml/aasee.kml", bbox=True)
     assert "bbox" in result
+    assert "crs" in result
     assert result["bbox"] == pytest.approx([7.594213, 51.942465, 7.618246, 51.957278], abs=tolerance)
+    assert result["crs"] == "4326"
+
+
+@pytest.mark.skipif(sys.platform == "darwin", reason="MacOS does not load the file properly")
+def test_kml_extract_tbox():
```

Comment:
KML time extraction was implemented by #99.
This test is included to properly close #103